main.css

```diff
@@ -0,0 +1,195 @@
+body {
+  background-color: hsl(204, 43%, 93%);
+  height: 100vh;
+  font-family: Karla;
+  margin: 0;
+}
+
+.main-container {
+  display: flex;
+  justify-content: center;
+  min-height: 100vh;
+}
+
+.main-div-align-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.main-div {
+  max-width: 636px;
+  border-radius: 16px;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  box-shadow: 0px 0px 20px rgb(0 0 0 / 20%);
+}
+
+.intro {
+  justify-content: center;
+  background-color: white;
+  padding: 0 40px;
+  height: 216px;
+}
+
+.main-text {
+  margin-top: 38px;
+  font-size: 24px;
+  color: #2ab2af;
+}
+
+.second-heading {
+  color: #c0df34;
+  font-size: 18px;
+  margin: 24px 0px 0px;
+}
+
+.third-heading {
+  color: #96a6bd;
+  font-size: 15px;
+  margin: 10px 0px 0px;
+  line-height: 26px;
+}
+
+.sub-part {
+  display: flex;
+  flex-direction: row;
+}
+
+.description {
+  display: flex;
+  flex-direction: column;
+  background-color: #2ab2af;
+  flex: 1;
+  padding: 40px;
+}
+
+.description-title {
+  font-size: 18px;
+  color: #e5eff5;
+  margin: 0;
+}
+
+.price-main {
+  display: flex;
+  flex-direction: row;
+  margin: 16px 0px 0px;
+  gap: 20px;
+}
+
+.price {
+  font-size: 30px;
+  color: #e5eff5;
+  margin: 0;
+}
+
+.per {
+  font-size: 16px;
+  margin: 8px 0px 0px;
+  color: #e5eff5;
+}
+
+.full-access {
+  margin: 8px 0px 0px;
+  font-size: 16px;
+  color: #e5eff5;
+}
+
+.sign-up-button {
+  width: 100%;
+  padding: 17px;
+  margin: 30px 0px 0px;
+  font-size: 14px;
+  font-family: Arial, Helvetica, sans-serif;
+  color: white;
+  background-color: #c0df34;
+  border: none;
+  border-radius: 8px;
+  cursor: pointer;
+  font-weight: 600;
+  box-shadow: 0px 0px 10px rgb(0 0 0 / 20%);
+}
+
+.sign-up-button:hover {
+  background-color: #2fc6c4;
+}
+
+.about {
+  flex: 1;
+  background-color: #2fc6c4;
+  padding: 40px;
+  display: flex;
+  flex-direction: column;
+}
+
+.why-us {
+  font-size: 18px;
+  color: #e5eff5;
+  margin: 0;
+}
+
+.list-style {
+  list-style: none;
+  margin: 17px 0px 0px;
+  padding: 0;
+  color: #e5eff5;
+  font-size: 14px;
+  line-height: 20px;
+}
+
+li {
+  position: relative;
+}
+
+li::before {
+  position: absolute;
+  content: '';
+  width: 15px;
+  height: 15px;
+  background-image: url(./assets/javascript.svg);
+  border-radius: 50%;
+  left: -2rem;
+}
+
+li::after {
+  content: ',';
+}
+
+.login {
+  padding: 40px;
+}
+
+.login-text {
+  margin: 0;
+  color: #c0df34;
+}
+
+.input-main-div {
+  position: relative;
+  margin: 8px 0px 0px;
+  padding: 10px 0px 10px 0px;
+  width: 40%;
+  border: 2px solid;
+  border-radius: 8px;
+  font-size: 16px;
+  background-color: #c0df34;
+}
+
+.input-field {
+  border: none;
+  outline: none;
+  background-color: transparent;
+  font-size: large;
+  width: 75%;
+}
+
+.search-image {
+  position: absolute;
+  height: 40px;
+  width: 40px;
+  left: 178px;
```

Comment:
use property: right 
instead of left, so that its easy to manage, and also scaleable if the input width is changed